qc/workflow_exome.cwl

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Exome QC workflow"
+requirements:
+    - class: SubworkflowFeatureRequirement
+inputs:
+    bam:
+        type: File
+        secondaryFiles: [^.bai]
```

Comment:
[This alignment output](https://github.com/susannasiebert/arvados_trial/blob/1c90d18ef4d5c6bb06c5b2db45d997cfb1a3e9dc/unaligned_bam_to_bqsr/workflow.cwl#L32) shows the bam index as being `.bai` rather than `^.bai`.